doc/release-notes.md

```diff
@@ -28,7 +80,8 @@ data
     ├── pbta-snv-mutect2.vep.maf.gz
     ├── pbta-snv-strelka2.vep.maf.gz
     ├── pbta-sv-lumpy.tsv.gz
-    └── pbta-sv-manta.tsv.gz
+    ├── pbta-sv-manta.tsv.gz
+    └── README.md
```

Comment:
@yuankunzhu this essentially reverts a change introduced in #107. Can you remove this reference to a README please? I would fix myself and merge but I can't push to this branch.